api/src/db.js

```diff
@@ -38,7 +38,7 @@ sequelize.models = Object.fromEntries(capsEntries);
 
 // En sequelize.models están todos los modelos importados como propiedades
 // Para relacionarlos hacemos un destructuring
-const { Product, Review, User, Brand, Type, Storage, Address, Ram, Cart, Order, Product_cart } = sequelize.models;
+const { Product, Review, User, Brand, Type, Storage, Address, Ram, Cart, Order, Product_cart, Product_order } = sequelize.models;
```

Comment:
Debajo, en las relaciones entre tablas, se debe definir de esta forma:
Product.belongsToMany(Order, {through: Product_order}); //N to N relation stablished
Order.belongsToMany(Product, {through: Product_order'});
Funciona en este momento pero creo que solo por el mero hecho de que el string que está en through coicide justo con el nombre de la tabla Product_order.

Reply:
LIsto, ya envio los cambios